src/site/markdown/model_formats.md

```diff
@@ -1753,14 +1753,14 @@ needs to be trained on the same VW version that is included in the `aloha-vw-jni
     <td>N / A</td>
   </tr>
   <tr>
```

Comment:
These links weren't working on the site page.  I'm not sure if this will fix the issue.  @deaktator please let me know if you think this will work.